scripts/check-if-e2e-tests-should-run-on-ci.sh

```diff
@@ -0,0 +1,20 @@
+# Since the e2e tests take a while to run and it could present an inconvenience
+# to be making sure the e2e tests work on every single PR, only run the e2e
+# tests on CI for PRs to master or on commits directly to dev or master
+if [[ "$GITHUB_BASE_REF_SLUG" = "master" ]]; then
+  echo "SHOULD_RUN_E2E=true" >> $GITHUB_ENV && export SHOULD_RUN_E2E=true
+  echo 'Will run E2E tests because this is a PR to master'
+else
+  if [[ "$GITHUB_REPOSITORY" = "ibi-group/datatools-server" ]] && [[ "$GITHUB_REF_SLUG" = "master" || "$GITHUB_REF_SLUG" = "dev" || "$GITHUB_REF_SLUG" = "github-actions" ]]; then
```

Comment:
Can the two `if` blocks be combined? (The action is the same in both)

Reply:
The echoed message isn't the same though, so I'd prefer to keep them separate.